app/SearchApi/SearchApi.Web/Notifications/WebHookNotifierSearchEventStatus.cs

```diff
@@ -118,7 +118,7 @@ private async Task ProcessEvents(string searchRequestKey, PersonSearchAdapterEve
 
         private async Task ProcessWaveSearch(string searchRequestKey, string eventName,string dataPartner )
         {
-            if (!EventName.Finalized.Equals(eventName))
+            if (!EventName.Finalized.Equals(eventName) || !EventName.InformationReceived.Equals(eventName))
```

Comment:
is this correct?

Reply:
I believe so, the goal is to skip processing for wave if it's just information or it's finalize.. What are your thoughts ?

Reply:
ok